ci-operator/config/openshift/loki/openshift-loki-main.yaml

```diff
@@ -38,6 +38,11 @@ images:
 - context_dir: operator
   dockerfile_path: Dockerfile
   from: base
+  inputs:
+    ocp_builder_rhel-9-golang-1.20-openshift-4.14:
+      as:
+      - golang:1.20.4
+      - gcr.io/distroless/static:nonroot
```

Comment:
We should use the 4.14 base image w/o the builder for the runner FROM directive. See https://github.com/openshift/loki/blob/main/Dockerfile.ocp#L7